<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cwImportWallsDialog</class>
 <widget class="QDialog" name="cwImportWallsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="FileLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>21</y>
     <width>774</width>
     <height>13</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>490</y>
     <width>177</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="WarningLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>774</width>
     <height>283</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QTreeView" name="WallsTreeView"/>
   <widget class="QWidget" name="layoutWidget_2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="WallsErrorsWidget">
        <attribute name="title">
         <string>Survex Errors</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QListView" name="WallsErrorListView"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="ExporErrorsWidget">
        <attribute name="title">
         <string>Import Errors</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="ImportButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>470</y>
     <width>77</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <zorder>FileLabel</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>splitter</zorder>
  <zorder>ImportButton</zorder>
  <zorder>ImportButton</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>WarningLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
